ui/app/styles/core/message.scss

```diff
@@ -1,38 +1,98 @@
 .message {
-  &.is-list {
-    margin: $size-10 0;
+  background: $blue-010;
+  border: 1px solid $blue-100;
+  margin-bottom: $spacing-s;
+  padding: $spacing-s $spacing-m $spacing-m $spacing-s;
+  position: relative;
+
+  .message-icon {
+    color: $blue;
+    margin-right: $spacing-xs;
+  }
+
+  .close-button {
+    background: transparent;
+    border: 0;
+    color: $grey;
+    cursor: pointer;
+    position: absolute;
+    right: $spacing-s;
+    top: $spacing-m;
+  }
+
+  .message-title {
+    color: $blue-500;
+    font-size: 16px;
+    font-weight: $font-weight-bold;
+    line-height: 1.25;
+  }
+
+  .close-button + .message-title {
+    padding-right: $spacing-m;
+  }
+
+  .message-body {
+    border: 0;
+    margin-top: $spacing-xxs;
   }
 
-  &.is-warning {
-    .message-body {
-      color: $yellow-darkest;
-      padding: 0.75rem 1.25rem;
+  p {
+    font-size: $size-8;
+    border: 0;
+    padding: 0;
+  }
+
+  .message-actions {
+    margin-top: $spacing-xs;
+
+    a,
+    a:not(.button):not(.file-delete-button):not(.tag) {
+      color: $blue;
+      font-weight: $font-weight-semibold;
+      text-decoration: none;
+    }
+
+    > * + * {
```

Comment:
would this select all children of `a` elements and adjacent siblings?

Reply:
Yep, immediate children and their adjacent siblings, no matter what they are

Reply:
I didn't word that right. It's actually just the siblings (so it doesn't affect the first child or if there's only one)

Reply:
ah! good to know!